lib/src/controllers/reader_mode_controller.dart

```diff
@@ -159,18 +158,25 @@ class ReaderModeController extends ChangeNotifier {
   }
 }
 
-class _UriWithScrollPosition {
+/// Tuple class containing the [position] for a given [uri].
+class UriWithScrollPosition {
+  /// The url that was loaded
   final Uri uri;
```

Comment:
Just to be sure.. in this case, if we still want `UriWithScrollPosition` to be private, we can simply put it in a different file and not export that file, right?

Reply:
same with `AncestorWithLevel`, `RowsAndColumns`, `ProcessedHtml` and`GrabArticleResult`

Reply:
yes, but it's open source, if people want to override certain API elements, they can actually do that now, so maybe they are best kept public throughout.